packages/apps/plugins/plugin-sheet/src/components/Sheet/Sheet.stories.tsx

```diff
@@ -51,7 +54,42 @@ const Story = ({ cells, ...props }: SheetProps & { cells?: Record<string, CellSc
     return null;
   }
 
-  return <SheetComponent {...props} sheet={sheet} />;
+  return (
+    <Tooltip.Provider>
+      <Sheet.Root {...props} sheet={sheet}>
+        <SheetToolbar />
+        <Sheet.Grid />
+        <Sheet.StatusBar />
+        <Sheet.Debug />
+      </Sheet.Root>
+    </Tooltip.Provider>
+  );
+};
+
+const SheetToolbar = () => {
+  const { selected, setFormat } = useSheetContext();
+  const handleAction: ToolbarProps['onAction'] = ({ type }) => {
+    const styles: Record<string, string> = {
+      left: 'text-left',
+      center: 'text-center',
+      right: 'text-right',
+      bold: 'font-bold',
+      highlight: 'bg-blue-100/25 border-yellow-100',
+    };
```

Comment:
Define constants outside of render functions.

Reply:
yeah, this is a very first cut hack. i'm not sure how we want to do this. i've tried to use some of the componentry and logic from the editor toolbar. lots of possibilities to factor out.